src/lib/rest/rest.cpp

```diff
@@ -1042,6 +993,58 @@ static int connectionTreat
   }
 
   // 3. Finally, serve the request (unless an error has occurred)
+  // 
+  // URL and headers checks are delayed to third MHD call 
+  //
+  
+  if (urlCheck(ciP, ciP->url) == false)
+  {
+    LM_W(("Bad Input (error in URI path)"));
+    restReply(ciP, ciP->answer);
+  }
+
+  ciP->servicePath = ciP->httpHeaders.servicePath;
+
+  if (servicePathSplit(ciP) != 0)
+  {
+    LM_W(("Bad Input (error in ServicePath http-header)"));
+    restReply(ciP, ciP->answer);
+  }
+
+  if (contentTypeCheck(ciP) != 0)
+  {
+    LM_W(("Bad Input (invalid mime-type in Content-Type http-header)"));
+    restReply(ciP, ciP->answer);
+  }
+  else if (outFormatCheck(ciP) != 0)
+  {
+    LM_W(("Bad Input (invalid mime-type in Accept http-header)"));
+    restReply(ciP, ciP->answer);
+  }
+  else
+    ciP->inFormat = formatParse(ciP->httpHeaders.contentType, NULL);
+
+  if (ciP->httpStatusCode != SccOk)
+  {
+    LM_W(("Bad Input (error in URI parameters)"));
+    restReply(ciP, ciP->answer);
+    return MHD_YES;
+  }
+  LM_T(LmtUriParams, ("notifyFormat: '%s'", ciP->uriParam[URI_PARAM_NOTIFY_FORMAT].c_str()));
+
+  // Set default mime-type for notifications
+  if (ciP->uriParam[URI_PARAM_NOTIFY_FORMAT] == "")
+  {
+    if (ciP->outFormat == XML)
```

Comment:
@Fermin, not easy to know exactly what you refer to ... We still have no written style-guide ...
Now,
All if-else statements inside {}, also one-liners.
That's all I can see right here. Anything else?